examples/js/postprocessing/EffectComposer.js

```diff
@@ -187,7 +206,7 @@ THREE.Pass = function () {
 	// if set to true, the pass clears its buffer before rendering
 	this.clear = false;
 
-	// if set to true, the result of the pass is rendered to screen
+	// if set to true, the result of the pass is rendered to screen. This is set automatically by EffectComposer.
```

Comment:
Should `Pass.renderToScreen` be private, then?

Reply:
At least one example uses Pass without using EffectComposer and needs to set this. Maybe it would be better to make this private but then we'd have to at least refactor that example first.

Reply:
That example is `webgl_postprocessing_unreal_bloom.html`.